client/signup/config/steps.js

```diff
@@ -142,16 +142,24 @@ export default {
 		providesDependencies: [ 'themeSlugWithRepo' ]
 	},
 
-	// Currently, this step explicitly submits other steps to skip them, and
+	// Currently, these two steps explicitly submit other steps to skip them, and
 	// should not be used outside of the `domain-first` flow.
 	'site-or-domain': {
 		stepName: 'site-or-domain',
-		apiRequestFunction: stepActions.createSiteOrDomain,
 		props: {
 			headerText: i18n.translate( 'Choose how you want to use your domain.' ),
 			subHeaderText: i18n.translate( "Don't worry you can easily add a site later if you're not ready." )
 		},
+		providesDependencies: [ 'designType', 'siteId', 'siteSlug', 'domainItem', 'themeSlugWithRepo' ],
+	},
+	'site-picker': {
+		stepName: 'site-picker',
+		apiRequestFunction: stepActions.createSiteOrDomain,
+		props: {
+			headerText: i18n.translate( 'Choose your site?' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Choose a site' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).